### ISSUE TITLE
ESH RS16 316 Endevender RF PH

### ISSUE THREAD

**Issue by sveip** [nilskaa/Smartthings#1]
Have you tried the ESH RS16 316 Endevender RF PH? 

http://proff.elko.no/elko-smart-home-316-endevender/esh-rs16-316-endevender-rf-ph-article2628-1780.html

**Comment by nilskaa** [comment]
Haven't tried yet. Paring devices direction to each other in smartthings is a problem, but want to try setting this up as a separate device that can control other devices through smartthings. Only reason for not prioritising this yet is that it Probably Won't work very well due to the time delay from performing action to propagsting down into other device. Would be interesting testing this though 😊

**Comment by sveip** [comment]
Thanks for your replay. I was thinking about the scenario you propose, controlling though ST. If The delay is below 200ms I think it will be quite usable. Do you have any experience on delays on ST?

The motivation to use this dimmer is the aesthetics and operational "feel", not necessarily to use it together with the dimmer in the same series :)

**Comment by nilskaa** [comment]
Agreed, Could use it to drive fibaro dimmer for instance.  However i have experienced delay of ~2 sec in a similar scenario - where I have 2 light switches connected through ST with fibaro switch behind the physical switches.

**Comment by nilskaa** [comment]
Hi, I have not followed this up myself. I have however been told that someone managed to pair (as you did) and tweek a dh to submit events upon pressing the button. Then picking this up in WebCore to switch on/off something. The delay was apparently only 1-2 seconds and acceptible. It should not be hard to write a DH for this and perhaps map the on/off events to an on/off cluster to avoid the WebCore part. I still have not acquired the remote dimmer, so can't do much to help here yet.

**Comment by torandreroland** [comment]
I opened up a pull request for the endevender device handler. @nilskaa can you consider merging it?
